src/sync/installation.ts

```diff
@@ -227,7 +228,7 @@ async function doProcessInstallation(app, job, installationId: number, jiraHost:
 		logger
 	);
 
-	const newGithub = new GitHubClient(installationId, logger);
+	const newGithub = new GitHubClient(getCloudInstallationId(installationId), logger);
```

Comment:
The only changes in the code that's using the GitHub client is that we now pass in an `InstallationId` object instead of a numeric installation ID, so that the client has all the information it needs to call the cloud or on-premise GitHub API.